doc/api/crypto.md

```diff
@@ -3383,7 +3383,10 @@ comparing HMAC digests or secret values like authentication cookies or
 [capability urls](https://www.w3.org/TR/capability-urls/).
 
 `a` and `b` must both be `Buffer`s, `TypedArray`s, or `DataView`s, and they
-must have the same length.
+must have the same byte length.
+
+If at least one of `a` and `b` is a `TypedArray`, the result may depend on
+the platform byte order.
```

Comment:
`may depend` sounds like this isn’t deterministic, but I assume it is? It’s always going to depend on the platform byte order, right?

Reply:
Good point! My thinking was that the result of some inputs (e.g., `timingSafeEqual(new Uint8Array([1, 1]), new Uint16Array([0x0101]))`) does not depend on the platform byte order, but that's probably not what others would take away from the way I worded it.

Reply:
Yeah, I see …  maybe I would say something like ``If at least one of `a` and `b` is a `TypedArray` with more than one byte per entry, like `Uint16Array`, the result will be computed using platform byte order``?

Reply:
Sorry about the delay, fixed!